articles/libraries/auth0-swift/index.md

```diff
@@ -0,0 +1,307 @@
+---
+description: How to install, initilize and use Auth0.Swift
+url: /libraries/auth0-swift
+---
+
+# Auth0.swift
+
+Auth0.swift is a client-side library for [Auth0](http://auth0.com).
+
+## Requirements
+
+iOS 9+ and Xcode 8 (Swift 3.0)
+
+::: panel-info Swift 2.3
+For Swift 2.3 you need to use [v1@swift-2.3](https://github.com/auth0/Auth0.swift/tree/v1@swift-2.3) branch.
+:::
+
+If using CocoaPods we recommend version 1.1.0 or later.
+
+## Installation
+
+### CocoaPods
+
+Auth0.swift is available through [CocoaPods](http://cocoapods.org). 
+To install it, simply add the following line to your Podfile:
+
+```ruby
+pod "Auth0", '1.0.0'
+```
+
+### Carthage
+
+In your Cartfile add this line
+
+```
+github "auth0/Auth0.swift" "1.0.0"
```

Comment:
I think we should recommend any 1.0 compatible version, so 

`github "auth0/Auth0.swift" ~> 1.0`